src/ai/rewardfunction/StepRewardFunction.java

```diff
@@ -0,0 +1,31 @@
+package ai.rewardfunction;
+
+import rts.GameState;
+import rts.TraceEntry;
+import rts.UnitAction;
+import rts.units.Unit;
+import util.Pair;
+
+public class StepRewardFunction extends RewardFunctionInterface {

```

Comment:
Could you explain how this reward function gets used in the `mine/lava minigame`? Do player gets a -1 reward for stepping into the Lava?

Reply:
This function is returning the total number of steps overall. Another reward function for stepping on a mine/lava would be a good idea. Right now if a unit steps on a mine/lava cell they instantly get removed.